examples/evm-multichain-dapp/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```

Comment:
This file should be in `.gitignore`, please remove.

Reply:
Added ` .gitignore` file